tutorials/prometheus-adapter.md

```diff
@@ -193,23 +192,44 @@ use a PostgreSQL docker image that has both Pg_Prometheus and TimescaleDB
 extensions installed: https://hub.docker.com/r/timescale/pg_prometheus.
 
 ```bash
-docker run --network prometheus_timescale_network  --name pg_prometheus -d -p 5432:5432 timescale/pg_prometheus:master postgres \
+docker run --network prometheus_timescale_network  --name pg_prometheus -d -p 5432:5432 timescale/pg_prometheus:latest postgres \
      -csynchronous_commit=off
 ```
 
 Note that this image inherits from the official PostgreSQL image and so all
 options documented there are applicable to this image as well. This is
 especially important for users that wish to persist data outside of docker
-volumes is the PGDATA environment variable and accompanying volume mount (visit https://hub.docker.com/_/postgres/ for available configuration options)
+volumes is the PGDATA environment variable and accompanying volume mount (visit https://hub.docker.com/\_/postgres/ for available configuration options)
```

Comment:
typo in http address

Reply:
What is the typo?

Reply:
\ was entered to fix something in the preview parsing engine, link works fine without it though so we'll remove it.